cmd/ingester/app/flags.go

```diff
@@ -114,7 +114,12 @@ func AddOTELFlags(flagSet *flag.FlagSet) {
 	flagSet.String(
 		KafkaConsumerConfigPrefix+SuffixEncoding,
 		DefaultEncoding,
-		fmt.Sprintf(`The encoding of spans ("%s") consumed from kafka`, strings.Join(kafka.AllEncodings, "\", \"")))
+		fmt.Sprintf(`The encoding of spans ("%s") consumed from kafka`, strings.Join(
+			append(kafka.AllEncodings,
+				kafka.EncodingZipkinJSON,
```

Comment:
I think these options would need to be qualified to indicate only supported with the otel image.